audio/speech/getting-started/get_started_with_chirp_2_sdk.ipynb

```diff
@@ -0,0 +1,1331 @@
+        "    streaming_config=streaming_config,\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "h1d508ScfD9I"
+      },
+      "source": [
+        "#### Run the streaming recognition request\n",
+        "\n",
+        "Finally, you are able to run the streaming recognition request."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "QCq-iROpfl9t"
+      },
+      "outputs": [],
+      "source": [
+        "def requests(config: cloud_speech.RecognitionConfig, s: list) -> list:\n",
+        "    yield config\n",
+        "    yield from s\n",
+        "\n",
+        "\n",
+        "response = client.streaming_recognize(requests=requests(stream_request, audio_requests))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "d__QUGkWCkGh"
+      },
+      "source": [
+        "Here you use a helper function to visualize transcriptions and the associated streams."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "_qWA8jXYuMH3"
+      },
+      "outputs": [],
+      "source": [
+        "streaming_recognize_results = parse_streaming_recognize_response(response)\n",
+        "streaming_recognize_output = get_recognize_output(\n",
+        "    short_audio_sample_bytes, streaming_recognize_results\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "agk_M0xRwzv0"
+      },
+      "outputs": [],
+      "source": [
+        "for audio_sample_bytes, transcription in streaming_recognize_output:\n",
+        "    print_transcription(audio_sample_bytes, transcription)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oYCgDay2hAgB"
+      },
+      "source": [
+        "### Perform real-time speech recognition"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "F83r9aiNhAgD"
+      },
+      "source": [
+        "#### Define real-time recognition configuration\n",
+        "\n",
+        "As for the streaming transcription, you define the real-time recognition configuration which allows you to set the model to use, language code of the audio and more."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "j0WprZ-phAgD"
+      },
+      "outputs": [],
+      "source": [
+        "real_time_config = cloud_speech.RecognitionConfig(\n",
+        "    language_codes=[\"en-US\"],\n",
+        "    model=\"chirp_2\",\n",
+        "    features=cloud_speech.RecognitionFeatures(\n",
+        "        enable_automatic_punctuation=True,\n",
+        "    ),\n",
+        "    auto_decoding_config=cloud_speech.AutoDetectDecodingConfig(),\n",
+        ")"
```

Comment:
Is it possible to reuse this config across the different modes? Maybe define it first, then input into the different wrapper classes like `StreamingRecognitionConfig`.

I also wonder if it would make sense to present real-time (online) recognition first since it's a little simpler.

Reply:
It would make sense. But I want to make it as much clear as possible.